plugins/PerObjectSettingsTool/__init__.py

```diff
@@ -11,15 +11,15 @@
 def getMetaData():
     return {
         "plugin": {
-            "name": i18n_catalog.i18nc("@label", "Per Object Settings Tool"),
+            "name": i18n_catalog.i18nc("@label", "Per Model Settings Tool"),
```

Comment:
Please forgive me if I still type Per Object Settings Tool once in a while :wink: